### PR TITLE
Bookstore - Initialize

### PR DESCRIPTION
**Pull request by maov19**
## Added features:

- Setup a new React application using Create React App and React Router
- Created a `/components` directory for your reusable components: 
- Add 2 routes in your application:
1. `/ `
> Displays a book list, where each book has a title and author
> Includes a form that allows for creating a new book

2. `/categories` 
> Displays a button that checks for a status

-  A navigation bar including both components was created.

## Demo:
bookstore-initialize

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,148 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Martin Ossa**
+- GitHub: [@maov19]
+
```
Comment (Yerimah) [inline README.md]: - [ ] You have done an excellent job implementing the features for this project. I would kindly suggest you add the correct GitHub and LinkedIn links to your Author section of the README document and ensure it is clickable.

**Review by Yerimah — CHANGES_REQUESTED** [review]
# STATUS: REQUEST CHANGES ♻️  

Hi @maov19, 👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f4d634a5755566852566e493378656f516f6d2f67697068792e676966


## Highlights
- Descriptive PR summary and title ✔️ 
- Git flow ✔️ 
- No linter error ✔️ 
- Meaningful commits ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (maov19) [reply README.md]: thanks @Yerimah! the requested changes were already added 🔧 

bookstore-initialize-review1

**Comment by maov19** [comment]
Dear reviewer, 

The issues observed by the previous reviewer are already corrected here✔️ 
Please let me know if there is anything else I should work on.

> thanks @Yerimah! the requested changes were already added 🔧
> 
> bookstore-initialize-review1

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @maov19, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**